apps/web/src/components/Basenames/UsernameProfileSectionBadges/Badges/index.tsx

```diff
@@ -153,7 +153,7 @@ export function BadgeImage({ badge, claimed, score, size, name }: BadgeImageProp
   const showTalentScore = Boolean(claimed && score && badge === 'TALENT_SCORE');
 
   return (
-    <div className="group relative flex h-[160px] w-[160px] items-center justify-center">
```

Comment:
I would expect removal of these values to have a significant effect on layout. Double checking this has been tested on big viewports too?

Reply:
yes desktop & mobile